src/pages/Dashboard/sub-pages/Vaults/VaultsTable/index.tsx

```diff
@@ -140,7 +140,10 @@ const VaultsTable = (): JSX.Element => {
       {
         Header: t('locked_btc'),
         accessor: 'lockedBTC',
-        classNames: ['text-right']
+        classNames: ['text-right'],
+        Cell: function FormattedCell({ value }: { value: BitcoinAmount }) {
+          return <>{displayMonetaryAmount(value)}</>;
```

Comment:
L145L: Is the fragment needed? Can't we return just the string (result of `displayMonetaryAmount` function)?